runtime/autoload/provider/clipboard.vim

```diff
@@ -31,34 +31,51 @@ function! s:try_cmd(cmd, ...)
 endfunction
 
 let s:cache_enabled = 1
-if executable('pbcopy')
-  let s:copy['+'] = 'pbcopy'
-  let s:paste['+'] = 'pbpaste'
-  let s:copy['*'] = s:copy['+']
-  let s:paste['*'] = s:paste['+']
-  let s:cache_enabled = 0
-elseif exists('$DISPLAY') && executable('xsel')
-  let s:copy['+'] = 'xsel --nodetach -i -b'
-  let s:paste['+'] = 'xsel -o -b'
-  let s:copy['*'] = 'xsel --nodetach -i -p'
-  let s:paste['*'] = 'xsel -o -p'
-elseif exists('$DISPLAY') && executable('xclip')
-  let s:copy['+'] = 'xclip -quiet -i -selection clipboard'
-  let s:paste['+'] = 'xclip -o -selection clipboard'
-  let s:copy['*'] = 'xclip -quiet -i -selection primary'
-  let s:paste['*'] = 'xclip -o -selection primary'
-elseif executable('lemonade')
-  let s:copy['+'] = 'lemonade copy'
-  let s:paste['+'] = 'lemonade paste'
-  let s:copy['*'] = 'lemonade copy'
-  let s:paste['*'] = 'lemonade paste'
-elseif executable('doitclient')
-  let s:copy['+'] = 'doitclient wclip'
-  let s:paste['+'] = 'doitclient wclip -r'
-  let s:copy['*'] = s:copy['+']
-  let s:paste['*'] = s:paste['+']
-else
-  echom 'clipboard: No clipboard tool available. See :help clipboard'
+let s:err = ''
+
+function! provider#clipboard#Error() abort
+  return s:err
+endfunction
```

Comment:
Yea, the only way to see that error would be by calling `provider#clipboard#Error()`, which does not happen at the moment.